blocks/library/button/index.js

```diff
@@ -32,7 +32,8 @@ registerBlock( 'core/button', {
 	attributes: {
 		url: attr( 'a', 'href' ),
 		title: attr( 'a', 'title' ),
-		text: children( 'a' )
+		text: children( 'a' ),
+		align: metadata()
```

Comment:
>That was the first API I had, I made it optional in a second step. I thought it'd be handy to avoid duplication, but I'm ok reverting to a mandatory attribute name.

To avoid errors, it's probably better to keep as-is.